pegasus/sites.v3/code.org/public/teacher-dashboard/nav.haml

```diff
@@ -6,11 +6,16 @@ content-type: text/ng-template
 
 #nav{"ng-show"=>"section.id"}
   #goback{style: "float:left"}
-    %a{href: '/teacher-dashboard#/'}= I18n.t('dashboard_navigation_home_page')
-    %span{style: "opacity:0.5"} &nbsp; &#9654; &nbsp;
-    %a{href: '/teacher-dashboard#/sections'}= I18n.t('dashboard_navigation_all_sections')
-    %span{style: "opacity:0.5"} &nbsp; &#9654; &nbsp;
-    %b{style: "color: #ff8b02;"}= I18n.t('dashboard_navigation_section', section_name: '{{section.name}}')
+    %div{'ng-show' => "new_flow()"}
+      %span{style: "color: #00adbc; font-size: 18px; font-weight: 'bold';"} &nbsp; &#8249; &nbsp;
+      %a{"ng-href" => CDO.studio_url("/home/#classroom-sections"), style:"color:#00adbc;font-size:14px;font-family:'Gotham 4r';font-weight:'bold';"}='View All Sections'
```

Comment:
This "View All Sections" text needs to be localized.  You'll need to add it to the Pegasus i18n gsheet (I sent you a link) which will get translated and synced, and then use it here.